nav/nav.css

```diff
@@ -0,0 +1,69 @@
+.container{
+    padding: 1rem;
+    gap: 2rem;
+}
+.component-container{
+    padding: 2rem 4rem 2rem 2rem;
+}
+.component-details{
+    padding: 1rem;
+}
+.nav-examples{
+    padding: 1rem;
+    margin-bottom: 2rem;
+}
+.navbar2{
+    padding: 0.5rem;
+    box-shadow: 2px 5px 10px rgba(194, 192, 192, 0.7);
```

Comment:
one suggestion would be to change color from rgba to hex(#) like other colors used in your CL.